.github/workflows/push.yml

```
@@ -0,0 +1,31 @@
+name: Push Workflow for develop-postgres
+
+on:
```

yamllint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)